test/services/notifications/setup/download-recipients.service.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, before } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const SessionHelper = require('../../../support/helpers/session.helper.js')
+
+// Thing under test
+const DownloadRecipientsService = require('../../../../app/services/notifications/setup/download-recipients.service.js')
+
+describe('Notifications Setup - Download recipients service', () => {
+  let session
+  const referenceCode = 'RINV-00R1MQ'
```

Comment:
NIT - We try to place `const`s before `let`s separated by a new line.

```suggestion
  const referenceCode = 'RINV-00R1MQ'

  let session  
```